### ISSUE TITLE
Emit descriptions in all target languages

### ISSUE THREAD

**Issue by schani** [quicktype/quicktype#521]
quicktype now supports types and class property descriptions, which should be emitted as comments or doc strings in the generated code. 
 [See how it's done in Go](https://github.com/quicktype/quicktype/blob/d2ede2aa52ca2bbf9829094b3c852aa2b28f99b3/src/Language/Golang.ts#L177).

- [x] C#
- [x] Swift
- [x] TypeScript
- [x] Rust
- [x] Java
- [x] C++
- [x] Objective-C
- [ ] JSON Schema
- [x] Elm
- [x] Simple types

**Comment by schani** [comment]
@krk This shouldn't be more than a few lines of code, in case you're interested.

**Comment by Dr-Emann** [comment]
1. Do we also want to do anything with `title`s as well?
2. As a note, for rust, the comments should be done with doc comments (triple slash), so that the types/fields will get docs based on the description provided.

**Comment by schani** [comment]
@Dr-Emann We already use `title` to name the types.

**Comment by schani** [comment]
JSON Schema is now held up by input rather than output, i.e. the JSON Schema input doesn't completely pull in all the descriptions, or something happens in between.

Elm is held up by Elm: https://github.com/elm-lang/elm-compiler/issues/1678#issuecomment-366843853

**Comment by schani** [comment]
Elm is done, too.  I'm closing this and will open a separate issue for JSON Schema.
